templates/zerver/why-zulip.md

```diff
@@ -20,6 +20,10 @@ keyboard shortcuts, and blazingly fast clients make Zulip a daily
 pleasure. Usability is important; if people don’t like a communications
 tool, they just won't use it.
 
+<div class="illustration">
+<img src="../../static/images/comparison.png" alt="" class="ilu-left"/>
```

Comment:
* Didn't realise the path would work like that, change it & the filename!
* Not having the `alt` tag triggers the `Images should have a text alternative or presentational role` ARIA tool error. Maybe `role="presentation"` would fit it better?